### DISCUSSION TITLE
PieceInfo in PDF

### DISCUSSION THREAD

**Discussion by ThomasJochum** [QuestPDF/QuestPDF#518] [General]
Does QuestPDF support encoding of PieceInfo in the PDF?

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. We apologize for the delayed response.

PieceInfo encoding is not currently supported by QuestPDF and is not on our roadmap.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
